packages/qwik/src/core/qrl/qrl-class.ts

```diff
@@ -35,6 +36,7 @@ export interface QRLInternalMethods<TYPE> {
   getHash(): string;
   getCaptured(): any[] | null;
   getFn(
+    maybeAsync?: boolean,
```

Comment:
If you put this as the last argument, you don't have to pass it if it's false

Reply:
I'll refactor for this